packages/shared/merge-options.js

```diff
@@ -16,12 +16,29 @@ function getStubs (optionStubs, config) {
   }
 }
 
+function getMocks (optionMocks, config) {
```

Comment:
This is basically the same as the getStubs function, can you split it into a generic getOptions function?

Reply:
Sure. Maybe we can use it to provide an interface to all options.

Reply:
Ok! done.

- `methods` also works in the same way - just need to add it to `mergeOptions`. Should we add that in this commit? It's only a few lines.
- `computed` might need a bit more work, since it needs a `getter` and `setter` to be set up.
- I did not try `props` and `data` yet.

Reply:
Great work! Yes, please add methods :)